src/vec_bcd_ppc.h

```diff
@@ -147,55 +155,39 @@ vec_bcdadd (vui32_t a, vui32_t b)
             : "v" (a),
               "v" (b)
             : "cr6" );
-#else
-#if 0
-        _Decimal128 d_t, d_a, d_b;
-        d_a = vec_BCD2DFP (a);
-        d_b = vec_BCD2DFP (b);
-        d_t = d_a + d_b;
 #else
         _Decimal128 d_t;
         d_t = vec_BCD2DFP (a) + vec_BCD2DFP (b);
-#endif
         t = vec_DFP2BCD(d_t);
 #endif
         return (t);
 }
 
-/** \brief Subtract two Vector Signed BCD 31 digit values.
```

Comment:
did you change the order of these functions? makes it tough to review.  I presume there were no code changes.